### ISSUE TITLE
Suggestion: use shairport-sync to receive airplay requests

### ISSUE THREAD

**Issue by hackerb9** [robobenklein/ubuntu-airplay#15]
I notice in your README that you say you can no longer test this as you don't have access to Apple equipment. You can install 'shairport' as a server on a Linux box hooked up to a TV or stereo. That's what I do so my guests can stream Spotify from their iPhones. It works great.

**Comment by robobenklein** [comment]
I used to do testing with the XBMC airplay receiver while that project was still going, but the primary goal of this application was to get the video backend and mirroring working on Linux.

As of right now, the best that this application can do is send a series of screenshots as a slideshow, which was nowhere near what I had hoped it would turn out to be. That worked well enough while I was using it at school (2+ years ago), but it seems that the image slide timeout duration was only supported on hardware receivers. (I couldn't get anything to work on XBMC since it kept images on the screen for a minimum of 1sec/each)

I haven't touched this repo since mostly because I haven't had the need. Once I got out of highschool I wasn't surrounded by Apple tech any more and it simply wasn't necessary for me. (And there was little-to-no attention on this repo back then either...)

Most of the hacking left to do on this project is to break down the Airplay H.264 encoding stream protocol so that we could ideally get native performance like originally intended. I'm actually not even sure that it's the same protocol at this point, it's been so long.

As for audio, that was the fairly simple part, so simple in fact I completely forgot about adding it into this repo and I just have a few shell scripts left over. For that I was using `pulseaudio-module-raop` which was available in the Ubuntu repos.

**Comment by hackerb9** [comment]
Ah, that explains why I wasn't able to use openairplay to actually play any music. It'd be nice if those scripts got added in to the project. Should I bother to submit more bug reports or is this project dead to you? (For example, the service browser looks for "_airplay._tcp.local", but shairport is advertising on "_raop._tcp.local").

**Comment by robobenklein** [comment]
The project is feature-dead from my perspective, I made this in order to solve a problem I had with technology at an Apple-certified highschool, but I got fed up enough with those tech restrictions that I'm glad not to have to deal with them any more. (Keep in mind this repo was purely video-stream oriented instead of audio-oriented.)

I've been keeping this repo up mostly for archival purposes, but I still will accept PRs or minor edits. As for the core goal - I don't really have any reason to continue this project. (I'd probably start from scratch even if I were to continue to maintain it, since it's been over 3 years since starting it and this code is relatively poor quality compared to my stuff now.)

If you are looking for an audio solution: https://github.com/hfujita/pulseaudio-raop2
